content/news/buzz/august-2024-status-update/contents.lr

```diff
@@ -0,0 +1,57 @@
+
+* We've begun work on updating Mobile Forge and related binary tooling to support the new official binary tag and framework formate required by the PEP-compliant patch for iOS support. So far, we're able to compile simple binary packages like ``lru-dict``, and non-Python dependencies like ``libjpeg``; we're working on restoring binary support for more complex binary dependencies.
+
+* We've landed the `Android test runner script <https://github.com/python/cpython/pull/121595>`__, fixed a number of issues that have arisen in CPython's Android support since the original patches landed, and `deployed an Android CPython buildbot <https://buildbot.python.org/#/builders/1594>`__. This buildbot has revealed some edge cases (primarily in socket and signal handling) that only appear to manifest under the unusual load conditions generated by a buildbot, so we've temporarily disabled the buildbot while we investigate the cause of these problems.
+
+* We completed work on `a new API for document-based apps <https://github.com/beeware/toga/pull/2666>`__. 
+
+* We completed work on `an API for status icon apps <https://github.com/beeware/toga/pull/2768>`__.
+
+* We released `Toga 0.4.6 <https://pypi.org/project/toga/0.4.6/>`__. This includes the support for document-based apps, status icon apps, and a number of other long-awaited features, such as apps without menubars, available in a stable release.
+
+* We `updated the support for WebView on Windows <https://github.com/beeware/toga/pull/2764>`__ to a more recent version, and `resolved an issue with compatibility with Windows Store Python <https://github.com/beeware/toga/pull/2778>`__. 
+
+* We diagnosed and contributed fixes for a number of issues in tools that BeeWare depends on for our build tooling. This includes updates to CI configurations to support Sphinx 8, `fixes for Sphinx 8 support in a Sphinx plugin <https://github.com/crate/sphinx_csv_filter/pull/49>`__, and `diagnosing issues with a recent Towncrier update <https://github.com/twisted/towncrier/issues/632>`__.
+
+* We presented a keynote at `Kiwi PyCon <https://kiwipycon.nz>`__ on the future of Python. 
+
+What's next?
+--------------
+
+In September, we'll be continuing work on binary package support for mobile platforms. Once we've restored support for binary wheels using the new binary tag and framework format, and added support for Python 3.13, we'll start upstreaming the changes we've made so that the wider Python ecosystem can start supporting iOS and Android. We'll also start working on the fixes to tools like ``pip``, ``cibuildwheel`` and ``meson-python`` that need to have explicit iOS support. We expect that this work will be ongoing for several months, but some of the work should be submitted upstream for review by the end of the month.
```

Comment:
```suggestion
In September, we'll be continuing work on binary package support for mobile platforms. Once we've restored support for binary wheels using the new binary tag and framework format, and added support for Python 3.13, we'll start upstreaming the changes we've made so that the wider Python ecosystem can start supporting iOS and Android. We'll also start working on the fixes to tools like ``pip``, ``cibuildwheel``, and ``meson-python`` that need to have explicit iOS support. We expect that this work will be ongoing for several months, but some of the work should be submitted upstream for review by the end of the month.
```